src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+    class="card card__image"
+    data-qa="card"
+  >
+      <div class="card__title">
+        <p class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__content">
+          <ul class="card__content-stars">
```

Comment:
Name should be `card__stars stars`, to define it as the _Block_ stars that is also child of the _Block_ card